package.json

```diff
@@ -129,10 +130,9 @@
     }
   },
   "lint-staged": {
-    "**/*.{js,jsx,ts,tsx}": [
-      "npm run lint",
-      "npm run test:ci",
-      "git add ."
-    ]
+    "*.{js,jsx,ts,tsx}": [
+      "eslint"
+    ],
+    "*.js": "eslint --cache"
```

Comment:
Is it possible for this config to be equal to the one from the components repo

```json
  "lint-staged": {
    "**/*.{js,jsx,ts,tsx}": [
      "npm run lint",
      "git add ."
    ]
  },
```

Reply:
@tehKapa I want you thoughts on this.